examples/angular/src/app/story/latest-ask-stories/latest-ask-stories.component.ts

```diff
@@ -0,0 +1,48 @@
+import {ChangeDetectionStrategy, ChangeDetectorRef, Component, OnInit} from '@angular/core';
+import { HackerNewsStory } from '../../../business-logic/domain/models/hacker-news-story.model';
+import {
+  LatestAskStoriesErrorViewState, LatestAskStoriesLoadedViewState,
+  LatestAskStoriesLoadingViewState,
+  LatestAskStoriesViewState
+} from "./latest-ask-stories.view-state";
+import {
+  GetHackerNewsLatestAskStoriesInteractor
+} from "../../../business-logic/domain/interactors/get-hacker-news-latest-ask-stories.interactor";
+import { NavigationService } from "../../services/navigation.service";
+
+@Component({
+  selector: 'app-latest-ask-stories',
+  templateUrl: './latest-ask-stories.component.html',
+  styleUrls: ['./latest-ask-stories.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class LatestAskStoriesComponent implements OnInit {
+  public viewState!: LatestAskStoriesViewState;
```

Comment:
**Nit:** Does it make sense to shorten this to `state`? Would it be confusing? For me better if it's terse, but then it might lose meaning.

Reply:
yes, if we are making this project as a sample, I rather being strict with the naming